src/cookieSyncManager.js

```diff
@@ -7,6 +7,10 @@ export default function cookieSyncManager(mpInstance) {
 
     // Public
     this.attemptCookieSync = function(previousMPID, mpid, mpidIsNotInCookies) {
+        if (!mpInstance._Persistence.isEnabled()) {
```

Comment:
looks like you can mark https://go.mparticle.com/work/SQDSDKS-6100 as done in this sprint also :)